_episodes_rmd/01-starting-with-data.Rmd

```diff
@@ -54,19 +56,38 @@ We want to:
 
 To do all that, we'll have to learn a little bit about programming.
 
-### Loading Data
+# Getting started
```

Comment:
Perhaps this should be ## here?
```suggestion
## Getting started
```

Otherwise the lower headings need to go up one level as well...